lib/DispatchMap.pm6

```diff
@@ -25,7 +29,12 @@ method !add-dispatch($ns,@key,$value) {
         } else {
             $_;
         }
-        nqp::bindattr($param,Parameter,'$!nominal_type',nqp::decont($nominal-type));
+        if $*RAKU.compiler.version >= v2020.10.104.g.1.f.090.e.04.d {
```

Comment:
This should probably still be $*PERL if you want to support older versions.

Reply:
`($*PERL // $*RAKU)` would be optimal, perhaps. And the check should actually be done once, in the global context.